var/spack/repos/builtin/packages/py-azure-cli/package.py

```diff
@@ -76,7 +76,7 @@ class PyAzureCli(PythonPackage):
     depends_on("py-azure-mgmt-recoveryservices@0.4.0:0.4", type=("build", "run"))
     depends_on("py-azure-mgmt-recoveryservicesbackup@0.6.0:0.6", type=("build", "run"))
     depends_on("py-azure-mgmt-redhatopenshift@0.1.0", type=("build", "run"))
-    depends_on("py-azure-mgmt-redis@7.0.0:7.0", type=("build", "run"))
+    depends_on("py-azure-mgmt-redis@7.0", type=("build", "run"))
```

Comment:
Can this now be `@7.0:`?

Reply:
it's actually an accidental change, i'll remove it.